src/Interfaces/DynamoTestStation.ts

```diff
@@ -0,0 +1,17 @@
+interface DynamoTestStation {
+  testStationId : string,
+  testStationAddress: string,
```

Comment:
I think we're missing `testStationAccessNotes`...

```suggestion
  testStationId : string,
  testStationAccessNotes: string,
  testStationAddress: string,
```

Reply:
Its empty so I'm wondering if dynamo will fill it in as null if its not there